src/org/usfirst/frc/team2706/robot/OI.java

```diff
@@ -48,19 +49,24 @@ public OI(Joystick driverStick, Joystick controlStick) {
         // Joystick for driving the robot around
         this.driverStick = driverStick;
 
+        // Joystick for controlling the mechanisms of the robot
+        this.controlStick = controlStick;
+
         // TODO we need to start using controlStick and not driverStick for non-testing buttons
 
         EJoystickButton backLeftButton = new EJoystickButton(driverStick, 5);
         backLeftButton.runWhileHeld(new AlignAndDistance(24));
-
+        
         EJoystickButton a = new EJoystickButton(driverStick, 1);
         a.runWhileHeld(new Climb());
 
         EJoystickButton b = new EJoystickButton(driverStick, 2);
         b.whenPressed(new GearHandlerToggle());
 
-        // Joystick for controlling the mechanisms of the robot
-        this.controlStick = controlStick;
+        // test the camera integration
+        EJoystickButton cameraButton = new EJoystickButton(driverStick, 3);
```

Comment:
Maybe tell us either in the name or the comment what button 3 is on the joystick?